### DISCUSSION TITLE
Match fetcher - workers

### DISCUSSION THREAD

**Discussion by jacob-gershon** [AlgoBet-Finance/algobet-interface#13] [Ideas]
@plutonous123 proposed to use Isports' API - a sports data supplier.
How to fetch the match data from their APIs?

**Comment by albert-57** [comment]
1. api Schedule & Results: get all matches in the WC.
2. api Team Profile: get information about teams (Team logo url. The picture is saved locally for use, please do not call it directly.???)
3. Worker: => get match id by matchtime(timestamp) 2 worker
3.1. Pre-match (cur timestamp < matchtime)  European Odds: update odds pre-match
3.2 In-play (cur timestamp >= matchtime) for each match requires:
  - Live Odds Changes: live update odds each match
  - worker to live update minute played
  - Livescores Changes: live update score for each match
